integration/integrate-with-email.adoc

```diff
@@ -0,0 +1,16 @@
+[id="integrate-using-email"]
+= Integrating with email
+include::modules/common-attributes.adoc[]
+:context: integrate-with-email
+
+toc::[]
+
+[role="_abstract"]
+You can use email as a notification method by forwarding alerts from {product-title} to a standard email provider.
+
+To forward alerts from the {product-title-short} platform to an email address, you can use either the `Default recipients` or `Deployment annotations` parameter.
```

Comment:
Why we use each would be useful here. We've also forgotten about namespace annotations. We should communicate why and what. 

You use default recipients to send to a standard and centralized team. You use deployment annotations to send to the specific team managing the deployment. You use namespace annotations to salably manage audiences by namespace.

Reply:
Here is a place where it would make sense to describe the contextual flow of how we decide who the audience is. 

Using an annotation key, you can define an audience to notify about policy violations associated with any given deployment or namespace. If the deployment has the annotation, its value overrides the default; otherwise, if the namespace has the annotation, its value overrides the default.

We define a key that tells ACS to look at for this audience in any annotation that has that key. 

1. If a deployment has an annotation key and audience we will send to the audience defined in that annotation key. 
2. If a deployment doesn't have an annotation key, we will check if the namespace has an annotation key and send to the audience defined there.
3. If no annotation keys are present we will send to the default recipient defined in the integration.